servers/src/test/java/tachyon/master/EditLogTest.java

```diff
@@ -0,0 +1,242 @@
+    Assert.assertEquals(parents, op.get("parents", new TypeReference<List<Integer>>() {}));
+    Assert.assertEquals(children, op.get("children", new TypeReference<List<Integer>>() {}));
+    Assert.assertEquals("fake command", op.getString("commandPrefix"));
+    Assert.assertEquals(data, op.getByteBufferList("data"));
+    Assert.assertEquals("Comment Test", op.getString("comment"));
+    Assert.assertEquals("Tachyon Examples", op.getString("framework"));
+    Assert.assertEquals("0.3", op.getString("frameworkVersion"));
+    Assert.assertEquals(DependencyType.Narrow, op.get("dependencyType", DependencyType.class));
+    Assert.assertEquals(1, op.getInt("dependencyId").intValue());
+    Assert.assertEquals(1409349750338L, op.getLong("creationTimeMs").longValue());
+  }
+
+  @Test
+  public void createFileTest() throws IOException {
+    // Write CREATE_FILE
+    TachyonURI createFilePath = new TachyonURI("/test/createFilePath");
+    mEditLog.createFile(true, createFilePath, false, 128L, 1409349750338L);
+    mEditLog.flush();
+    mEditLog.close();
+
+    // Read CREATE_FILE and check
+    EditLogOperation op = getSingleOpFromFile(mTmpPath);
+    Assert.assertEquals(101, op.mTransId);
+    Assert.assertEquals(EditLogOperationType.CREATE_FILE, op.mType);
+    Assert.assertTrue(op.getBoolean("recursive"));
+    Assert.assertEquals(createFilePath.toString(), op.getString("path"));
+    Assert.assertFalse(op.getBoolean("directory"));
+    Assert.assertEquals(128L, op.getLong("blockSizeByte").longValue());
+    Assert.assertEquals(1409349750338L, op.getLong("creationTimeMs").longValue());
+  }
+
+  @Test
+  public void createRawTableTest() throws IOException {
+    // Write CREATE_RAW_TABLE
+    ByteBuffer metadata = ByteBuffer.wrap(Base64.decodeBase64("BBBBBBBBBBBBB++"));
+    mEditLog.createRawTable(1, 2, metadata);
+    mEditLog.flush();
+    mEditLog.close();
+
+    // Read CREATE_RAW_TABLE and check
+    EditLogOperation op = getSingleOpFromFile(mTmpPath);
+    Assert.assertEquals(101, op.mTransId);
+    Assert.assertEquals(EditLogOperationType.CREATE_RAW_TABLE, op.mType);
+    Assert.assertEquals(1, op.getInt("tableId").intValue());
+    Assert.assertEquals(2, op.getInt("columns").intValue());
+    Assert.assertEquals(Base64.encodeBase64String(metadata.array()), op.getString("metadata"));
+  }
+
+  @Test
+  public void deleteTest() throws IOException {
+    // Write DELETE
+    mEditLog.delete(1, true, 1409349750338L);
+    mEditLog.flush();
+    mEditLog.close();
+
+    // Read DELETE and check
+    EditLogOperation op = getSingleOpFromFile(mTmpPath);
+    Assert.assertEquals(101, op.mTransId);
+    Assert.assertEquals(EditLogOperationType.DELETE, op.mType);
+    Assert.assertEquals(1, op.getInt("fileId").intValue());
+    Assert.assertTrue(op.getBoolean("recursive"));
+    Assert.assertEquals(1409349750338L, op.getLong("opTimeMs").longValue());
+  }
+
+  private EditLogOperation getSingleOpFromFile(String path) throws IOException {
```

Comment:
Usually we put test cases together, so could you move this util functions either before or after the normal test cases?